src/components/Header/Header.js

```diff
@@ -37,3 +39,9 @@ const Header = (props) => {
 };
 
 export default Header;
+
```

Comment:
I'd like to review all the propTypes together for understanding and areas of improvement.